tests/test_core.py

```diff
@@ -135,6 +135,19 @@ def test_points_from_xy():
     assert_geodataframe_equal(result, expected)
 
 
+def test_points_from_xy_with_crs():
+    latitude = [40.2, 66.3]
+    longitude = [-105.1, -38.2]
+    expected = geopandas.points_from_xy(longitude, latitude, crs="EPSG:4326")
+    df = pd.DataFrame({"longitude": longitude, "latitude": latitude})
+    ddf = dd.from_pandas(df, npartitions=2)
+    actual = dask_geopandas.points_from_xy(
+        ddf, "longitude", "latitude", crs="EPSG:4326"
+    )
+    assert isinstance(actual, dask_geopandas.GeoSeries)
+    assert list(actual) == list(expected)
```

Comment:
This won't in itself verify the that the crs is set, so maybe you can change this with `assert_geoseries_equal(actual.compute(), expected)`

Reply:
Updated in 88f03cf.